server/src/main/java/org/apache/druid/discovery/DiscoveryDruidNode.java

```diff
@@ -164,6 +179,12 @@ public DruidNode getDruidNode()
     return druidNode;
   }
 
+  @JsonProperty
+  public DateTime getStartTime()
+  {
+    return startTime;
+  }
```

Comment:
Why is startTime omitted while verifying the equality of this class?

Reply:
Since a DiscoveryDruidNode object is primarily identified by its DruidNode, role and the service map, I wanted to preserve the equality condition. Also, start_time might not be a concrete enough to decide the equality between two  DiscoveryDruidNode objects, if the other field values are the same. What do you think?

Reply:
Thanks for the explanation! I think it's better to keep it the current way.